lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word.nil?
+
+    ones = [1] * word.length
```

Comment:
This strategy made sense once I looked through the whole code base. I don't mind it as a strategy, but only with better naming that helps illustrate what it is actually doing and why its here.